### ISSUE TITLE
running on a local Windows machine

### ISSUE THREAD

**Issue by lonnietc** [go-micro/demo#7]
Hello,

I would like to run the demo on a local single Windows machine and not use any containers or cloud services.

How can this be done?
Thanks

**Comment by xpunch** [comment]
Kubernetes is only a way of service orchestration, of cource it can run without docker/kubernetes
1. Setup configurations for every service, like service address. You can simply edit config.go in every service, or replace env.NewSource with another source, like yaml/toml.
```
var cfg *Config = &Config{
	Address:               ":8090",
	AdService:             "localhost:9555",
}
```
for example, if ad service is running on :9555, frontend service should access it though localhost:9555.
3. Run every service with `go run`
4. Testing localhost:8090, default config is [config.go](https://github.com/go-micro/demo/blob/main/src/frontend/config/config.go)

**Comment by lonnietc** [comment]
Thanks for getting back to me and I will look into things a bit more.

Right now, I am also still trying to figure out how to get the Micro.exe up and running since I tried to follow a 2019 Blog article on it (https://blog.gopheracademy.com/advent-2019/building-a-microservices-network/) that did not work to start running a micro.exe node.

I also just found the Discord channel and am wondering if that is the best place to have discussions although it seems not to active at the moment?

Thanks again and have a great day
